CMakeLists.txt

```diff
@@ -500,6 +500,9 @@ if (LLAMA_CUBLAS)
 endif()
 
 if (WIN32)
+    if (MSVC)
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /bigobj")
```

Comment:
There should be a comment explaining why this is needed. Presumably object file size limitations?